src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -109,52 +123,60 @@ const FormDefinitionDetailPage = () => {
                   />
                 </CommonLabeledTextBlock>
                 <CommonLabeledTextBlock title='Status'>
-                  <FormStatusText identifer={formIdentifier} />
+                  <FormStatusText identifier={formIdentifier} />
                 </CommonLabeledTextBlock>
               </Stack>
             </CommonCard>
           </Grid>
           <Grid item xs={12} md={4}>
             <CommonCard title='Actions' titleComponent='h5'>
               <Stack gap={1.5}>
-                <RootPermissionsFilter permissions='canManageForms'>
-                  <ButtonLink
-                    to={generatePath(AdminDashboardRoutes.PREVIEW_FORM_DRAFT, {
-                      identifier: formIdentifier.identifier,
-                      formId: formIdentifier.draftVersion?.id || '',
-                    })}
-                    variant='contained'
-                    fullWidth
-                    disabled={!hasDraft}
-                  >
-                    Preview / Publish Draft
-                  </ButtonLink>
-                  <EditFormButton
-                    formIdentifier={formIdentifier}
-                    text={hasDraft ? 'Edit Draft' : 'New Draft'}
-                    editorType={FormEditorType.FormBuilder}
-                    variant='outlined'
-                  />
-                  <RootPermissionsFilter permissions='canAdministrateConfig'>
```

Comment:
Removing the JSON editor button from this panel and tucking it into the Form Builder was a [suggestion from Ash](https://www.figma.com/board/axGI9vwmuBRC4uZ11ZJ3RP/OP-HMIS-%2F-5792-%3A-Form-Builder-Workflow?node-id=1125-8863&t=AeD7zuvyJ9FM0tPz-0). It doesn't matter much since it won't be client-facing, but I went for it because I think it has the added advantage of removing some complexity from the `EditFormButton` above.